tests/devices/test_default_qutrit.py

```diff
@@ -0,0 +1,574 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane.plugin.DefaultQutrit` device.
+"""
+import math
+
+import pytest
+import pennylane as qml
+from pennylane import numpy as np, DeviceError
+from pennylane.devices.default_qutrit import DefaultQutrit
+from pennylane.wires import Wires, WireError
+
+OMEGA = np.exp(2 * np.pi * 1j / 3)
+
+
+U_thadamard_01 = np.multiply(
+    1 / np.sqrt(2),
+    np.array(
+        [[1, 1, 0], [1, -1, 0], [0, 0, np.sqrt(2)]],
+    ),
+)
+
+U_x_02 = np.array([[0, 0, 1], [0, 1, 0], [1, 0, 0]], dtype=np.complex128)
+
+U_z_12 = np.array([[1, 0, 0], [0, 1, 0], [0, 0, -1]], dtype=np.complex128)
+
+U_shift = np.array([[0, 0, 1], [1, 0, 0], [0, 1, 0]], dtype=np.complex128)
+
+U_clock = np.array([[1, 0, 0], [0, OMEGA, 0], [0, 0, OMEGA**2]])
+
+U_tswap = np.array(
+    [
+        [1, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 1, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 1, 0, 0],
+        [0, 1, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 1, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 1, 0],
+        [0, 0, 1, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 1, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 1],
+    ],
+    dtype=np.complex128,
+)
+
+U_tadd = np.array(
+    [
+        [1, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 1, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 1, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 1, 0, 0, 0],
+        [0, 0, 0, 1, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 1, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 1, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 1],
+        [0, 0, 0, 0, 0, 0, 1, 0, 0],
+    ],
+    dtype=np.complex128,
+)
+
+
+def include_inverses_with_test_data(test_data):
```

Comment:
What is this function doing and how/where is it used?

Reply:
I was following `tests/devices/test_default_qubit.py` very closely when I first started writing this test file. Since then, the two test files have diverged significantly due to the inherent differences between qubits and qutrits, and also due to the minimal number of operations available for `default.qutrit`. This function was added while referencing `test_default_qubit.py`, and I never ended up using it, so I'll just remove it. Thanks for pointing it out!